component-api/src/main/java/org/talend/sdk/component/api/record/Record.java

```diff
@@ -334,6 +335,10 @@ default Builder after(String entryName) {
 
         Builder updateEntryByName(String name, Schema.Entry schemaEntry);
 
+        Builder updateEntryByName(String name,
```

Comment:
In order to not break API other implementations (thinking at apic), please provide a `default` to this new method.